docs/Configuration/Overview.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Configuration Overview
+permalink: /configuration/overview
+category: Configuration
+menuOrder: 1
+---
+
+Cube.js is designed to work with different configuration sources. There
+are two main ways you can set configuration options; via a configuration
+file, commonly known as the `cube.js` file; and environment variables.
+
+There are some areas of overlap between these options, and we recommend
```

Comment:
@hassankhan it is a very interesting topic. I'm not sure that `cube.js` is the most preferable way right now because It lacks the environment management, i.e. the config in `cube.js` is global across all environments.
What do you think? I'm happy to move this convo to Slack.